runtime/compiler/control/CompilationThread.cpp

```diff
@@ -10371,6 +10371,11 @@ TR::CompilationInfo::compilationEnd(J9VMThread * vmThread, TR::IlGeneratorMethod
                   J9JITDataCacheHeader *exceptionTableCacheEntry = (J9JITDataCacheHeader *)((U_8 *)cacheEntry + aotMethodHeaderEntry->offsetToExceptionTable);
                   J9JITExceptionTable *metaData = (J9JITExceptionTable *) (exceptionTableCacheEntry + 1);
 
+                  if (!canRelocateMethod)
```

Comment:
If I understand the scenario correctly, these flags will never be seen in the metadata for a "live" compiled method. They serve only as a temporary hint passed to `addFreeBlock` to not print some problematic field. The only downside I see is occupying some bits in the `metaData->flags`.  If it's not too much hassle, maybe a we should add another parameter to `addFreeBlock`

Reply:
Yeah I'll take a look to do that.

Reply:
Adding a parameter is a bit cumbersome actually, because while `J9::CodeCacheManager::addFreeBlock` is only an OpenJ9 API, `J9::CodeCache::addFreeBlock` is an extensible API (ie there is a counterpart for this API in OMR), which means I'd have to also open an OMR PR.

Perhaps what's better is that I just add a single flag, namely `JIT_METADATA_IS_UNINITIALIZED`, and set that where we first allocate it, and reset it when we properly initialize the metadata.